README.md

```diff
@@ -255,6 +255,33 @@ Workbench schema lives in `api/db` --> all workbench related activities access/p
 
 CDR schema lives in `api/db-cdr` --> all cdr/cohort builder related activities access/persist data here
 
+## Generate cdr and public count databases for a CDR version
+
+This happens anytime a new cdr is released or if you want all the count data for databrowser and cohort builder generated locally. 
+
+### Generate count data in BigQuery from a cdr in biquery 
+`./project.rb generate-cdr-counts --bq-project all-of-us-ehr-dev \
```

Comment:
It's probably better to have stuff like "all-of-us-ehr-dev", "test_merge_dec26" and "20170130" as "BQ_PROJECT", "CDR_DATASET", "YYYYMMDD", etc... and then follow up with these as examples. Otherwise it's not clear where values are intended to be substituted. (Which looks like for everything?)

Reply:
How is it now?

Reply:
Yeah, that works.